.github/workflows/acr-build-publish.yml

```diff
@@ -0,0 +1,66 @@
+name: build and push to acr
+on: 
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to build'
+        required: true
+        default: 'development'
+        type: choice
+        options:
+          - development
```

Comment:
@Tatsinnit Yes right now all workflows are manual trigger except codeQL and binskim scan's which are also Github actions which run for every PR and it is still secured because it doesn't run if a first-time contributor creates a PR and maintainer has to approve and run them.